ddp-workspace/projects/ddp-osteo/src/assets/i18n/en.json

```diff
@@ -755,7 +755,7 @@
                                     "content": "“Are there any benefits that participants receive for participating?“"
                                 }
                             ],
-                            "To take part in these activities, please log into your dashboard to sign additional consent forms and provide updates to your/your child’s participation. If you have any questions, please reach out to the study team at info@osproject.org or 651-602-2020."
+                            "To take part in these activities, please log into your dashboard to sign additional consent forms and provide updates to your/your child’s participation. If you have any questions, please reach out to the study team at <a href=\"mailto:info@OSproject.org\" class='maincolor'>info@OSproject.org</a> or 651-602-2020."
```

Comment:
Is not anchor tag already blue?

Reply:
no it's not, this is a reason why I add class for color